interactions:
- request:
    body: '{"biller_id": 40, "account_number": "501000000004"}'
    headers:
      Accept: [application/vnd.regalii.v3.1+json]
      Accept-Encoding: ['gzip, deflate']
      Authorization: ['APIAuth 88879c1b066dc9aea6201f27be2bbba9:RaDsGKUzcfMd1G+dUR2az5b0Hqs=']
      Connection: [keep-alive]
      Content-Length: ['51']
      Content-MD5: [mDRNhwgEiKn41ooFxmHjFQ==]
      Content-Type: [application/json]
      Date: ['Thu, 03 Jan 2019 22:21:38 GMT']
      User-Agent: [python-requests/2.21.0]
    method: POST
    uri: https://api.casiregalii.com/bills
  response:
    body: {string: '{"code":"R2","message":"Invalid Account Number","id":8389}'}
    headers:
      Cache-Control: [no-cache]
      Connection: [keep-alive]
      Content-Type: [application/json; charset=utf-8]
      Date: ['Thu, 03 Jan 2019 22:21:40 GMT']
      Strict-Transport-Security: [max-age=31536000]
      Transfer-Encoding: [chunked]
      X-Content-Type-Options: [nosniff]
      X-Frame-Options: [SAMEORIGIN]
      X-Request-Id: [3881c285-365a-4f70-a0f6-61cc7d4f2790]
      X-Runtime: ['1.054893']
      X-XSS-Protection: [1; mode=block]
    status: {code: 422, message: Unprocessable Entity}
version: 1
